Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -170,7 +172,7 @@ def create_incidents(demisto_user: dict, incidents: list) -> dict:
     return data
 
 
-def process_incident_create_message(demisto_user: dict, message: str) -> str:
+def process_incident_create_message(demisto_user: dict, message: str) -> str:   # pragma: no cover
```

Comment:
Why did you add `# pragma: no cover` on this function?

Reply:
same as above